CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-## Contributor's Guide
```

Comment:
I had to normalize these headings, because Sphinx has certain requirements: the main heading must be level 1, and sub-headings cannot jump by more than 1.